crates/python/src/py_sync.rs

```diff
@@ -19,9 +19,14 @@
 /// assert say_hello("Rigetti") == "hello Rigetti"
 /// ```
 macro_rules! py_sync {
-    ($body: expr) => {{
+    ($py: ident, $body: expr) => {{
         let runtime = ::pyo3_asyncio::tokio::get_runtime();
         let handle = runtime.spawn($body);
+        let delay = ::std::time::Duration::from_millis(100);
```

Comment:
Any particular methodology that led to landing on 100ms here? It's fine if not, but I'm curious in what a good value for this is, or if it is really needed.